slowquant/qiskit_interface/linear_response/naive.py

```diff
@@ -30,7 +30,7 @@
 class quantumLR(quantumLRBaseClass):
     def run(
         self,
-        do_rdm: bool = True,
+        do_rdm: bool = False,
```

Comment:
The default option should be True?

Reply:
Yes and no. 
From quantum ressource perspective, False is better, as it leads to less overhead.
From classic ressource perspective (going beyond minimal basis) ,True would be better indeed.

Reply:
Simulating the RDMs is very sparse in the amount of Pauli Strings. So it just leads to more device calls

Reply:
Maybe it would be a good idea to re-order the LR algo. Start with GG and Gq elements, then simulate RDMs, then do qq. The simulation of RDMs might at this point be "for free" in terms of quantum computing ressources.

Reply:
I think we should have do_rdm default as true.
It seems like the more sane default option.

Reply:
I will just put it true in a commit where I implement an idea how to make the RDMs device call efficiently